app/cloud/index.html

```diff
@@ -0,0 +1,71 @@
+---
+title: Kong Cloud
+layout: default
+hide_subscribe: true
+id: cloud
+---
+
+<section class="section enterprise-section">
+  <div class="container">
+    <div class="row">
+      <div class="one-half column">
+        <h2>The Power of Kong, now delivered SaaS</h2>
+        <p>Kong Cloud comes with high-performance hosting, SLA, Backups, HA, 24x7 Support and more.</p>
```

Comment:
Originally, "Kong Cloud comes with high performances, SLA, Backups..". Is my version okay?